elasticsearch/templates/statefulset.yaml

```diff
@@ -78,6 +78,8 @@ spec:
           secret:
             secretName: {{ .name }}
         {{- end }}
+        - name: config
+          emptyDir: {}
```

Comment:
Why should we store the configuration here instead of regenerating it at each start?